xlive/Util/Hooks/Hook.cpp

```diff
@@ -84,46 +84,46 @@ void RetourClassFunc(BYTE *src, BYTE *restore, const unsigned int len)
 	);
 }
 
-void *VTableFunction(void *ClassPtr, DWORD index)
+void *VTableFunction(void *ClassPtr, size_t index)
 {
 	void **pVtable = *(void***)ClassPtr;
 	return pVtable[index];
 }
 
-void WriteBytes(DWORD destAddress, LPVOID bytesToWrite, const unsigned int numBytes)
+void WriteBytes(size_t destAddress, LPVOID bytesToWrite, const unsigned int numBytes)
 {
 	VirtualProtectAndExecutePatch((LPVOID)destAddress, numBytes, PAGE_EXECUTE_READWRITE,
 
 	memcpy((LPVOID)destAddress, bytesToWrite, numBytes);
 	);
 }
 
-void PatchCall(DWORD call_addr, DWORD new_function_ptr) {
-	DWORD callRelative = new_function_ptr - (call_addr + 5);
+void PatchCall(size_t call_addr, size_t new_function_ptr) {
```

Comment:
?

Reply:
using uintptr_t instead as we discussed before